src/liboslexec/llvm_util.cpp

```diff
@@ -1677,8 +1681,9 @@ LLVM_Util::setup_optimization_passes (int optlevel, bool target_host)
         builder.DisableUnrollLoops = false;
         builder.SLPVectorize = false;
         builder.LoopVectorize = false;
+#if OSL_LLVM_VERSION < 140
         builder.DisableTailCalls = false;
```

Comment:
That option was not actually hooked up to anything in LLVM since 2015 -- so you could also just remove that line entirely (we probably don't support LLVMs that old anymore anyway). The removal in LLVM 14 is just a simple cleanup, it didn't change any functionality.

Reply:
will do